pkg/sst/sst.go

```diff
@@ -234,3 +271,163 @@ func getBits(val, i, j uint32) uint32 {
 func isBitSet(val, n uint32) bool {
 	return val&(1<<n) != 0
 }
+
+func setBit(val, n uint32) uint32 {
+	return val | (1 << n)
+}
+
+func clearBit(val, n uint32) uint32 {
+	return val &^ (1 << n)
+}
+
+func setBFStatus(info *SstPackageInfo, status bool) error {
+	rsp, err := sendMboxCmd(info.pkg.cpus[0], CONFIG_TDP, CONFIG_TDP_GET_TDP_CONTROL, uint32(info.PPCurrentLevel))
+	if err != nil {
+		return fmt.Errorf("failed to read SST status: %v", err)
```

Comment:
nit: better to use `%w` when wrapping errors with `fmt.Errorf()`

```suggestion
		return fmt.Errorf("failed to read SST status: %w", err)
```

Ditto other similar `fmt.Errorf`